Cargo.toml

```diff
@@ -25,11 +25,9 @@ log = { version = "0.4.16" }
 env_logger = { version = "0.7.1" }
 colored = { version = "2.0.0" }
 
-rusb = { version = "0.9.1", optional = true }
-libusb1-sys = { version = "0.6.2", optional = true }
+rusb = { version = "0.9.3", optional = true }
+libusb1-sys = { version = "0.6.5", optional = true }
 
 [features]
-default = ["static", "vendored"]
-static = ["rusty_libimobiledevice/static", "plist_plus/static"]
-usb = ["libusb1-sys", "rusb"]
-vendored = ["rusty_libimobiledevice/vendored", "plist_plus/vendored"]
+usb = [ "libusb1-sys", "rusb" ]
+default = ["rusb", "zeroconf" ]
```

Comment:
We shouldn't be adding rusb to the default features, it's not even implemented

Reply:
rusb replaced for libusb1-sys in default.
commit 57b380b98e4b8552bf96a655921ad88fbd9a2268